### PR TITLE
CLOUD-3091 - Update builds to not use cacher

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-3091
Signed-off-by: Ken Wills <kwills@redhat.com>

### CODE REVIEW

File: jboss-eap-cd-openshift-modules/module.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ execute:
   user: '185'
 
 artifacts:
-- path: txn-recovery-marker-jdbc-common-1.1.2.Final-redhat-00001.jar
+- name: txn-recovery-marker-jdbc-common-1.1.2.Final-redhat-00001.jar
```
Comment (goldmann) [inline jboss-eap-cd-openshift-modules/module.yaml]: I suggest to use something like this:
```yaml
- name: txn-recovery-marker-jdbc-common
  target: txn-recovery-marker-jdbc-common-1.1.2.Final-redhat-00001.jar
  md5: cd68ad886a759d21f8dd0cb7646601ee

Same applies to other artifacts.
```

Comment (luck3y) [reply jboss-eap-cd-openshift-modules/module.yaml]: Ah, I see, so it can be overriden via name. Clever :)

Comment (goldmann) [reply jboss-eap-cd-openshift-modules/module.yaml]: Exactly!

**Comment by luck3y** [comment]
@bstansberry this PR should be all set I think, not sure what the other issues might be from reading the jira.

**Comment by luck3y** [comment]
FYI, this is causing issues for folks building images, someone approve it please :)

**Pull request merged**